src/camvidlog2/common/nms/np.py

```
@@ -0,0 +1,64 @@
+    inter_y2 = np.minimum(box[3], boxes[:, 3])
+
+    inter_w = np.maximum(0, inter_x2 - inter_x1)
+    inter_h = np.maximum(0, inter_y2 - inter_y1)
+    inter_area = inter_w * inter_h
+
+    area_box = (box[2] - box[0]) * (box[3] - box[1])
+    area_boxes = (boxes[:, 2] - boxes[:, 0]) * (boxes[:, 3] - boxes[:, 1])
+
+    union_area = area_box + area_boxes - inter_area
+    iou = np.where(union_area > 0, inter_area / union_area, 0.0)
+    return iou
```

Ruff (0.11.9)

Unnecessary assignment to `iou` before `return` statement

Remove unnecessary assignment

(RET504)